docs/build/avalanchego-apis/info.md

```diff
@@ -232,6 +232,59 @@ curl -X POST --data '{
 }
 ```
 
+
+### info.getVMs
+
+Get the virtual machines installed on this node.
+
+```cpp
+info.getVMs() -> {
+    vms: map[string][]string
+}
+```
+
+#### **Example Call**
+
+```bash
+curl -X POST --data '{
+    "jsonrpc":"2.0",
+    "id"     :1,
+    "method" :"info.getVMs",
+    "params" :{}
+}' -H 'content-type:application/json;' 127.0.0.1:9650/ext/info
+```
+
+#### **Example Response**
+
+```json
+{
+    "jsonrpc": "2.0",
+    "result": {
+        "vms": {
+            "jvYyfQTxGMJLuGWa55kdP2p2zSUYsQ5Raupu4TW34ZAUBAbtq": [
+                "avm"
+            ],
+                "mgj786NP7uDwBCcq6YwThhaN8FLyybkCa4zBWTQbNgmK6k9A6": [
+                "evm"
+            ],
+                "qd2U4HDWUvMrVUeTcCHp6xH3Qpnn1XbU5MDdnBoiifFqvgXwT": [
+                "nftfx"
+            ],
+                "rWhpuQPF1kb72esV2momhMuTYGkEb1oL29pt2EBXWmSy4kxnT": [
+                "platform"
+            ],
+                "rXJsCSEYXg2TehWxCEEGj6JU2PWKTkd6cBdNLjoe2SpsKD9cy": [
+                "propertyfx"
+            ],
+                "spdxUxVJQbX85MGxMHbKw1sHxMnSqJ3QBzDyDYEP3h6TLuxqQ": [
+                "secp256k1fx"
+            ]
+        }
```

Comment:
```suggestion
            "jvYyfQTxGMJLuGWa55kdP2p2zSUYsQ5Raupu4TW34ZAUBAbtq": [
                "avm"
            ],
            "mgj786NP7uDwBCcq6YwThhaN8FLyybkCa4zBWTQbNgmK6k9A6": [
                "evm"
            ],
            "qd2U4HDWUvMrVUeTcCHp6xH3Qpnn1XbU5MDdnBoiifFqvgXwT": [
                "nftfx"
            ],
            "rWhpuQPF1kb72esV2momhMuTYGkEb1oL29pt2EBXWmSy4kxnT": [
                "platform"
            ],
            "rXJsCSEYXg2TehWxCEEGj6JU2PWKTkd6cBdNLjoe2SpsKD9cy": [
                "propertyfx"
            ],
            "spdxUxVJQbX85MGxMHbKw1sHxMnSqJ3QBzDyDYEP3h6TLuxqQ": [
                "secp256k1fx"
            ]
        }
```
Is this correct? Feel free to discard if not right.

Reply:
My IDE autoformats this weirdly. Updated.